app/src/main/java/app/revanced/integrations/patches/components/AdsFilter.java

```diff
@@ -60,7 +60,8 @@ public AdsFilter() {
 
         final var medicalPanel = new StringFilterGroup(
                 SettingsEnum.HIDE_MEDICAL_PANELS,
-                "medical_panel"
+                "medical_panel",
+                "shorts_info_panel_overview"
```

Comment:
I don't think this component is soley used for medical information but information about the short in general, thus should be moved.

Reply:
I have seen the info panel link to Wikipedia once on normal videos.

Reply:
> Or maybe this should be part of the existing HIDE_HIDE_INFO_PANELS? I think that might be better.

Imagine the following scenario:

The user watches Shorts and notices the info panel, so they check out ReVanced patches and notice a patch that can hide components of Shorts; now, the user includes the patch, but hiding the info panel (a shorts component) is not present. For that reason, both patches should somehow be able to toggle this component on or off to satisfy the current patches' semantics as best as possible.